README.md

```diff
@@ -1,2 +1,144 @@
+- [📝 License](#license)
+
+# 📖 Personal Portfolio <a name="about-project"></a>
+
+Hello! Here you will see my first capstone project, based on an original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+## 🛠 Built With HTML & CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">GitHub</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.microverse.org/"></a>Microverse</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [production of main pages] **[home page and about page]**
+- [making page responsive] **[@media integration]**
+- [adding dynamically made javaScript section] **[featured speakers section]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Loom Walkthrough <a name="loom-walkthrough"></a>
+
+- [Loom Walkthrough Video Link](https://www.loom.com/share/cc776fd0ece644139437f5d3f6156331)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- install git, install VSCode
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/everinurmind/capstone-project
+```
+
+### Install
+
+- npm install
+
+
+### Usage
+
+- open with live server extention on VSCode
+
+
+### Run tests
+
+- There's no test for this project
+
+### Deployment
+
+The project has been deployed on (https://everinurmind.github.io)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+
+👤 **Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/everinurmind)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [developing personal portfolio] **[Adding more pages]**
+- [will make own design for portfolio] **[Creating own design]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/everinurmind/capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please leave me review if you have any questions or remarks.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to Microverse team, for all help and support on this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The [Creative Commons License](https://creativecommons.org/licenses/) requires that we give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly acknowledge her work as it is the legal and right thing to do 👍 💻